docs/guides/dataflow.md

```diff
@@ -119,7 +119,7 @@ Next, the `components` section contains third party services and local or extern
           "line_number": 36
         }
       ]
-    },
+    }
     // ...
```

Comment:
Maybe we want the commas to stay here? Not a big one though, I reckon.

Reply:
Trailing comma is invalid json so think its done right here.